### ISSUE TITLE
Test "test_set_of_ints_and_strs (tests.datatypes.test_set.SetTests)" is failing

### ISSUE THREAD

**Issue by johnnywell** [pybee/batavia#318]
```python
Traceback (most recent call last):
  File "/Users/johnny/Sandbox/Batavia/Tutorial/batavia/tests/datatypes/test_set.py", line 10, in test_set_of_ints_and_strs
    """, substitutions={"{1, '1'}": ["{'1': 1}"]})
  File "/Users/johnny/Sandbox/Batavia/Tutorial/batavia/tests/utils.py", line 550, in assertCodeExecution
    self.assertEqual(js_out, py_out, context)
AssertionError: "{1, '1'}\n" != "{'1', 1}\n"
- {1, '1'}
+ {'1', 1}
 : Function context
```

**Comment by martini97** [comment]
I think we already fixed that, try checking the code of https://github.com/pybee/batavia/pull/316

**Comment by glasnt** [comment]
Yup, this is fixed as of #316. Make sure you update your fork of the repo to the upstream master branch to get these changes ✨
